wavelink/ext/spotify/__init__.py

```diff
@@ -238,8 +238,7 @@ def __init__(self, data: dict[str, Any]) -> None:
         self.id: str = data['id']
         self.length: int = data['duration_ms']
         self.duration: int = self.length
-
-        self.isrc: str | None = data["external_ids"].get("isrc")
+        self.isrc: Optional[str] = data.get("external_ids", {}).get('irsc')
```

Comment:
Please change to 3.10 annotation

Reply:
I have updated the PR to reflect this request.